data/src/main/java/com/example/grosharies/data/groceryList/GroceryListRepository.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.grosharies.data.groceryList
+
+import androidx.lifecycle.LiveData
+
+class GroceryListRepository(private val groceryListDao: GroceryListDao) {
```

Comment:
So usually, in clean - repository pattern. 

You'd want to hide the implementation details. Often in a repository you have
- Remote source (for example consuming an api)
- Local source (for example Room)

We like to also hide the local source, for example by making an interface GroceryListDataSource
Then we can make (multiple) concrete implementations. For example
- RoomGroceryListDataSource
- xxxxxGroceryListDataSource

While the repository would impl the GroceryListDataSource

Reply:
(So you could easily switch from datasource in theory, not thinking about possible migrations of the local data)